zoom_monitor.py

```diff
@@ -198,7 +190,8 @@ def _open_participants_list(self):
                     # click implicitly creates a mouse-up on the first one,
                     # and that's the important part).
                     outer.click()
-                    outer.click()  # Channeling our inner grandma
+                    if not hovering:
+                        outer.click()  # Channeling our inner grandma
```

Comment:
Jessamy points out that this is confusing: amend the comment above to note that we only need to double-click it if we're not already hovering over it.

Reply:
:exploding_head: Ohh, maybe the part about hovering is that hovering over a button somehow changes it, and only the hovered version opens the list. When we start up and try clicking on the initial button, we haven't been hovering over it yet, which could explain why clicking does nothing (what it really does is change the class from Default to Hovering) and then clicking the second time actually does the thing we wanted. I wonder if there's a simpler way to do this, like "initially on startup, move the mouse to the Default button" and after that "ignore the Default, only care about the Hovering one, and always click just once." I also wonder whether figuring that out would be more work than it's worth.